cli/tests/unit/timers_test.ts

```diff
@@ -412,19 +412,6 @@ Deno.test(function clearTimeoutAndClearIntervalNotBeEquals() {
   assertNotEquals(clearTimeout, clearInterval);
 });
 
-Deno.test(async function timerMaxCpuBug() {
```

Comment:
I couldn't think of a good way to replicate this test without using metrics... thoughts?